### DISCUSSION TITLE
Main Quest/Mission?

### DISCUSSION THREAD

**Discussion by jdsutton7** [endless-sky/endless-sky#9873] [General]
I'm about 8 hours in, finding it difficult to track the main story. It's unclear what my objectives are besides flying around and grinding to buy larger ships.

Should a primary mission be added to give the player some direction? Being able to see a log of the main story would be helpful as well.

**Comment by Hecter94** [comment]
Right now, only one of the three main storylines is available, which means you can only get it in a specific region of space. Once the game is fully released, you'll be able to pick up the storyline from anywhere.

What have you done so far? You should be able to pick up the one currently available storyline by going into the southern part of the galaxy and checking spaceports; you'll get a few smaller quests, which will lead to a larger storyline. 
You can also access your player info panel by pressing `i`, which lets you access your logbook to see what you've done so far and get reminders of what you were asked to do next.

**Comment by jdsutton7** [comment]
So far I've explored almost the whole galaxy, built a small fleet, and
triggered one bit of story about nuclear terrorism.

I'll check the info panel, I've been using it but didn't notice the log
with all the info in there.

It would be really helpful to see any story objectives the same way I see
missions.

**Comment by Hecter94** [comment]
Make sure you're checking spaceports, specifically in the southwestern portion of human space, if you're interested in the main questline. 
You should be able to start picking up quests and storylines relatively quickly as long as you're moving around and checking the spaceports on various planets.

**Comment by xX-Dillinger-Xx** [comment]
Well what do you know. Another lost player. Luckily, this player came here looking for answers. They could have just as easily said this game is stupid and moved on to something else. I wonder how many people have done just that.

@jdsutton7
I'm just a player, like you, and have also felt the frustration with what you describe. We are not alone. There so many people complaining about the same thing, I wouldn't want guess the total number.
I have suggested a potential option. [Here](https://github.com/endless-sky/endless-sky/issues/9843#issuecomment-1959962407)
Unfortunately, there seems to be some real resistance to offering players 'in game' help. They claim "too much handholding ruins the experience". Well, ours, as well as soo many others experiences, and first impressions, where ruined because we couldn't figure out the point of the game. I feel that they forget that new players have no idea what this game is supposed to be about  I also agree that the logbook is useless.
This game is much more enjoyable the second time around. After you figure everything out it is actually a lot of fun. 
Hecter's answer was a little misleading. There are lots of other storylines outside of human space, once you find your way out of course. A few tips, visit the spaceport **_every_** time you land. Sometimes, things appear at random, or you may have arrived too early to continue your story. If you are asked to make a decision that feels important, it probably is and you have reached a fork in the story. Make a manual save, you may want to go back in time someday.

**Reply by Hecter94** [reply]
> Well, ours, as well as soo many others experiences, and first impressions, where ruined because we couldn't figure out the point of the game. I feel that either they forget that new players have no idea what this game is supposed to be about, or they just don't care.

I think this is looking at things a bit backward. The "point" of Endless Sky isn't the main storyline, managing a fleet, amassing wealth, getting a bunch of combat experience, or anything else that you can "miss." 
The "point" is to be a part of the world of Endless Sky. This may mean doing odd jobs, running trade routes,  building up your fleet, taking a side in conflict, just exploring, or a dozen other things. 

This means you may miss things, you may stumble into places you aren't ready for yet, and you may end up doing things very differently from what you expected to be doing, but whatever you end up doing will be *your* story, and **that** is the point of Endless Sky. 

The best experience is going in blind without any preconceived notions of what you *need* to be doing, what you *should* be doing, what the proper order to do things is, or how you should be doing things, just stumbling your way through the life of a starship captain and making your own story in the process. 
The worst experience is going online or reading through the files to figure out what you think you should be doing and trying to make sure you do everything in the proper order and not allowing yourself the opportunity to miss stuff.

Worrying about what the point of the game is, amusingly, missing the point of what the game is.

**Reply by xX-Dillinger-Xx** [reply]
Well then I guess most new players miss the "point."

**Reply by xX-Dillinger-Xx** [reply]
>The "point" of Endless Sky isn't the main storyline, managing a fleet, amassing wealth, getting a bunch of combat experience, or anything else that you can "miss." 

What? So your saying all the new players that get lost or don't know what to do, got the wrong idea of what ES is about? I would bet that the number of players that "got it wrong" is significantly higher then those that "got it right".
> The best experience is going in blind without any preconceived notions of what you need to be doing

So we are supposed to believe our character has decided to become a star ship cabin but has no preconceived notion of what to expect, or what's going on in the universe they live in? Apparently, our character has been saving for awhile, so we can assume that they would have done some homework. But, me as the player, I'm supposed to go in blind.  The doesn't really make a lot of sense.

>  The worst experience is going online or reading through the files to figure out what you think you should be doing

That is _exactly_ my point, and _exactly_ what is happening now.  You have created the "worst experience" by forcing players to do the very thing you said they shouldn't do.

This is a game, and players expect there to be some sort of reason for playing besides exploring someone's vision of space. Like others have pointed out this isn't a Simulator.

**Reply by Saugia** [reply]
This will be a long read, but I hope it clarifies what exactly Endless Sky is.

Some context as to why Endless Sky is the way it is might help to understand what is being said here by Hecter. ES is inspired by the Escape Velocity series, another top-down space game from the 90's that was rather popular in the day. Those games are similar in that they had tutorial missions at the start, and once through, leave the player open to the galaxy at large. Both main and side story would be missions that the player entirely stumble upon, and that was each EV's games indented experience. Hecter is right in saying that ES's experience is to be a part of an existing world, and that stumbling across certain aspects and story elements of the experience is how the game is to be played. I will say though that the main story is also a main part of the game.

However, there is no one way to play Endless Sky. Some players enjoy these types of space-trucker games as a pass-time. Some players enjoy making mass fleets and performing in combat. Some players like being immersed in the galactic level of a fictional universe. Some players of course want to play the main story. Some players don't want to be rushed to play the main story either. While the main story is a central part of the game, it is not the one-all intended experience of ES.

Going back to the EV series, all of this was purely random chance and running into these missions in spaceports, etc. (though as I mention, there was a "tour guide" in the form of checking spaceports, similar to what we have now). Now, did this mean that the overall community for the EV series decide that the game is about this or that? No, not really. It was about each individual player's experience. Did this hurt the game's success? Not at all. EV fit a niche that its audience enjoyed quite a lot with the proportion of those who did enjoy it outweighing those who did not.

Now let's take this back to ES. EV is an older game series, so it did have parts of it that may seem odd to how games are played nowadays. ES actually guides the player much more than EV did. The tutorial mission is a bit longer. In regards to guiding the player - when I said that we dislike hand-holding the player (this was a discord conversation), I was referring to the playthrough of a given story in the game. When it comes to basic principle, we already have measures to help guide the player to the central plot, ones that were something not present in the EV series. Within the last year, we've merged several missions that point to the facets of accepting the main story:

- There are a few missions that tell the player, both indirectly and directly, to check the spaceport.
- There was an addition of a few missions that direct the player southward to the current main story option (the Free Worlds).

The FW plot has a chance to start on several of the southern FW worlds in those spaceports. Though, the fact that the FW story is only one of three doors into the main plot is still far overlooked in these cases. Once the Republic side is in, a spaceport door will appear around the spaceports of Republic space. Once the Syndicate route is in, one will appear in Syndicate space. That will effectively cover the player's early game story access, and I don't doubt that some missions will be made to softly direct the player in those directions.

With that said, we already do have measures in the game to help direct the player to the story. We also are still a game in-progress, and there's much more coming that will aid with these measures. Ultimately though, this is a semi-emulation of the EV series, in that the world is to be stumbled upon through chance encounters. ES is not the type of game that has a quest list, a check list, a written-down point-to-point guide from every mission, or to the next mission. And yes, that is the intention of the game - that is the type of game Endless Sky is.

It is true that we have had complaints from players in the form of direction to the main story, and we understand what those players are saying. However, it is evident by the above that we already do have facets to help these cases in the game. Is how we choose to go about doing so subjective? Yes, for sure. Is it fair that some players disagree with the measures taken? Entirely. Is our vision of the game flawed, and has the community decided what the game is, and should we base it around that? I'm sorry to say, but this is not the case, and not by  far. 

These statements of large amounts of players having complaints, that so many people have issues with this, are not true to certain extents. It is true that some players do not find the story right away and it may take a minute or some guidance to find it outside of the game. It is true that there is a vocal few, and there may be a few who are not as vocal on it. However, it is not true that this is to an extent that it is such a detriment to the game as has been portrayed here. Endless Sky has been met with *overwhelmingly positive* feedback over the years. A lot of this is, to be expected, from the viewpoint of Escape Velocity players, which is a good portion of our audience. However, a large portion of our audience is also a fit niche of those who have very likely never heard of Escape Velocity. We don't follow the series to a T, and we certainly make our own unique take on the type of game. But it is without a doubt that we have a large and flourishing player base that appreciates the type of game we have envisioned and created. To say we refuse or ignore a wider audience in ignorance is to be ignorant to our actual audience that loves the game for what we are making it to be.

TL;DR: Endless Sky's vision is an open-ended story. It is not a directed story. It has main content, and large stories, and it does have directions and help to those stores. It is not out of the realm of possibility that we add indirect help to this content. However, it is intended to be that open-book - that random chance of falling into the world, and to not rush the player in doing so. The space-trucking, the combat, the fleet gameplay, each indeed are all one aspect of the game's intention - and that is exactly what the intention is. An open world. Ultimately, our vision may not be the best for some. But our vision is not something that is incomprehensible. It is not something that is ignorant. It is subjective, and at the end of the day, it is a matter of those who disagree with our vision, and those who enjoy it. If it is disagreeable, that is entirely fair, as is our desire to continue to produce what we wish to stick by. Those who enjoy it will continue to play it, and it may just be that Endless Sky isn't the game for those who disagree with it.

To wrap all this back to OP's post, they like the game overall. The game is sometimes confusing to new players, and this is entirely natural, and it's okay to reach out, and we do have methods of directing help. I don't believe further discussion is going to change much on the viewpoints stated here, but it is certainly something that has been heard, and we appreciate those willing to give their feedback.

**Reply by xX-Dillinger-Xx** [reply]
@jdsutton7 I'm sorry for hijacking your post. I shouldn't have tried to drag you into this. 
@Saugia Thank you for taking your time to respond in a calm and professional manner. Clearly, I didn't follow my own advice.
@Hecter94 and everyone else, I understand that some of my comments, where harsh, disrespectful, and inappropriate. For that, I apologize.

I'll keep it brief.
- I guess my biggest frustration comes from getting lost in a storyline. This usually happens during a 'time-out' in the story. What I would like to see is, better use of the logbook. It should include a better log of any storyline that I have already discovered and what I was asked to do next to continue where I left off. 
- I'm no longer suggesting to add anything that would be deemed a 'spoiler' within the actual game play. However, some subtle hints might be nice.
- after reflecting on what Saugia and others have mentioned, I now understand the value of not revealing too much up front. However, I would suggest using the player info area but on a separate tab. You could even label it "spoilers".(see  https://github.com/endless-sky/endless-sky/issues/9843#issuecomment-1959962407  for more details) Including the type of "help" I've been suggesting, in a hidden way, gives players the option to look at it or not. Players can use their own discretion.  By doing this, you are including a wider spectrum of players, while preserving your overall intent.  I feel, that currently, you are unintentionally excluding players that don't quite understand the concept of ES and just want a little help along the way.
- I am content with the acknowledgement that a little extra help may be needed in some areas and that it may be addressed in the future.

**Reply by Saugia** [reply]
Since it's relevant to this discussion, today we had a question asked during our monthly Campfire Chat regarding much of what has been said in this discussion and [replied to it](https://youtu.be/uj_6MVxssXI?t=3071) (it goes for a bit). I hope this helps explain our perspective on the situation.

**Comment by jdsutton7** [comment]
I don't have anything further to add, except that as a new player the
experience in fact was confusing. After exploring for 8 hours I felt I had
to search online to figure out where the content was in the game, and how
to access the content I had already stumbled upon. That said, I like the
game overall.

**Reply by xX-Dillinger-Xx** [reply]
Thanks for supporting my point.

**Reply by Hurleveur** [reply]
I played the game many years ago just massing ships in human space not knowing there was anything more to it until I came back more recently

**Reply by quyykk** [reply]
Me too. On my first save I had a huge fleet with Quarg tech before I realized there was a story lmao.

**Comment by RecursiveParadox** [comment]
Just an idea...we have in game timers for events. Perhaps if the player has not triggered the FW line by X amount of time, we open up the planets where the mission can trigger to include many others, say all of human space? I have no idea what the X should be.

This is just a suggestion. I personally believe that the way the game is now is perfect. This is because you only get to play ES for the first time, once. [Kinda. The developmental and open nature of the game are always throwing new things at First/Last.] But this issue has come up ever since 2015 on many different sites, so it's perhaps worth addressing.

**Reply by Saugia** [reply]
I feel like this goes back to having the main campaign complete for all sides eventually resolving the issue. Each part should only offer in certain areas of human space, and on special worlds like the current method. We don't want to ever have an exact pointer to where to get the mission - we do have missions that help guide the player in the right direction, and it's possible more may be added if it's still not enough, but we'd never go as to make it available everywhere, especially beyond the FW worlds it is intended to start on.

There would be direction in Republic space for the Republic Campaign, wherever that may be offered on. Same for Syndicate. So it wouldn't make sense to spread the start for one campaign if it meant overlapping others. Since the FW is still an opt-in, we wouldn't really want to try shoving it into the face of the player at every point, even if it's determined after a certain time frame.

**Reply by RecursiveParadox** [reply]
Well, the thing about that is that because life happens, we are no where near having the Republic nor Syndicate stories. So, until they are done and merged?
